packages/react-component-library/cypress/specs/RangeSliderE/index.spec.ts

```diff
@@ -1,6 +1,9 @@
-import { before, cy, describe, it } from 'local-cypress'
+/* eslint-disable jest/valid-expect */
```

Comment:
Would be good to add this to the list here: https://github.com/defencedigital/mod-uk-design-system/blob/master/packages/react-component-library/.eslintrc.js#L23